tests/Unit/NumericalAlgorithms/LinearSolver/ConjugateGradient/Test_ResidualMonitorActions.cpp

```diff
@@ -37,7 +41,7 @@ struct UpdateOperand;
 }  // namespace cg_detail
 }  // namespace LinearSolver
 
-using namespace ResidualMonitorActionsTestHelpers;
```

Comment:
Sorry, I take back what I said and I agree with you :) I missed that this is a test helpers namespace...